### PR TITLE
Add section on LXD profiles

### LINKED ISSUE #3115: [2.5] Mention support for charms referencing LXD profiles
As per https://github.com/juju/juju/pull/9317 and https://github.com/juju/juju/pull/9309

### PR DESCRIPTION
**Pull request by pmatulis**
resolves #3115

### CODE REVIEW

**Comment by pmatulis** [comment]
[Talk about LXD profiles](https://trello.com/c/c4vYB3Re/724-talk-about-lxd-profiles)

File: src/en/clouds-lxd-advanced.md [hunk 1]
```diff
@@ -214,6 +215,59 @@ Now that the cloud and credentials have been added the next step is to create a
...
+devices
+   unix-char
+   unix-block
+   gpu
+   usb
+```
+
+ - A profile is upgraded during the upgrade of the charm
+   (`juju upgrade-charm`).
+ - A profile is exposed at the machine level by using either the `show-machine`
```
Comment (manadart) [inline src/en/clouds-lxd-advanced.md]: "Profiles are displayed at the..." ?

File: src/en/clouds-lxd-advanced.md [hunk 2]
```diff
@@ -214,6 +215,59 @@ Now that the cloud and credentials have been added the next step is to create a
 controller. See [Creating a controller][clouds-lxd-creating-a-controller] on
 the main LXD page.
 
+## Charms and LXD profiles
+
+Juju (`v.2.5.0`) has the following functionality for those charms that make use
+of LXD profiles:
+
+ - A validity check is performed on the profile(s) during the deployment of the
```
Comment (manadart) [inline src/en/clouds-lxd-advanced.md]: Maybe the skinny on feature generally first? Something like:
"Juju (`v.2.5.0`) allows charms to include a LXD profile. The profile will be applied to a LXD container that the charm is deployed into."

File: src/en/clouds-lxd-advanced.md (hunk 1 above)
Comment (SimonRichardson) [reply src/en/clouds-lxd-advanced.md]: the alternative is using `juju status --format=yaml`

Comment (pmatulis) [reply src/en/clouds-lxd-advanced.md]: Thanks.

File: src/en/clouds-lxd-advanced.md (hunk 2 above)
Comment (pmatulis) [reply src/en/clouds-lxd-advanced.md]: Thanks.

**Pull request merged**